app/build.gradle

```diff
@@ -29,5 +29,6 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
Avoid using `+` in version numbers, it can lead to unpredictable builds. Get the latest version

Reply:
Noted. I will resolve this.